cinema/serializers.py

```diff
@@ -2,0 +1,103 @@
+from rest_framework import serializers
+
+from cinema.models import (
+    Genre,
+    Actor,
+    CinemaHall,
+    Movie,
+    MovieSession,
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.CharField(source="__str__", read_only=True)
+
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    capacity = serializers.SerializerMethodField(method_name="get_capacity")
+
+    class Meta:
+        model = CinemaHall
+        fields = ("id", "name", "rows", "seats_in_row", "capacity")
+
+    def get_capacity(self, obj):
+        return obj.capacity
```

Comment:
The method `get_capacity` is returning `obj.capacity`. If `capacity` is not a direct attribute of the `CinemaHall` model, this will raise an error. Consider calculating the capacity using `obj.rows * obj.seats_in_row` if `capacity` is meant to be derived from these fields.